scss/input/_theme.scss

```diff
@@ -50,6 +50,13 @@
             background-color: $accent;
             color: $accent-contrast;
         }
+
+        .k-state-disabled & {
+            &::selection {
+                @include appearance( "input" );
+                background-color: transparent;
```

Comment:
Seems more appropriate to use `color: $input-color` instead of `appearance( "input" )`, as the selection doesn't have a border and the background-color is overwritten